content/en/case-studies/booz-allen/index.html

```diff
@@ -4,7 +4,11 @@
 case_study_styles: true
 cid: caseStudies
 css: /css/case-studies-gradient.css
-featured: false
```

Comment:
```suggestion
featured: true
```
to be explicit about this(?)

Reply:
See below, `featured: true` is in line 8.